src/components/App/App.test.js

```diff
@@ -0,0 +1,12 @@
+import {render, screen} from '@testing-library/react'; 
+import React from 'react'; 
+import Navbar from '../Navbar/Navbar.component';
+
+
+describe('navbar', ()=>{
+    test('should contain a dark mode', ()=>{
+            render(<Navbar/>)            
+       const dark = screen.queryByText(/dark mode/i);            
+            expect(dark).toBeInTheDocument(); 
+    })    
+})
```

Comment:
Good test, 

Seems like there are spaces as indentation, you can use Eslint + Prettier (or just Eslint) in order to apply format in the code.